woocommerce/wc-admin
# Remove unused WC Pay feature flags

Continuing the work done in https://github.com/woocommerce/woocommerce-admin/pull/7233, this PR removes feature flags that are no longer of use, WC Pay ones in this case.

### Detailed test instructions:

1. Turn off marketplace suggestions so the remote config isn't fetched and the local hard coded values are drawn, `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com`.
2. Change your default country to one of the newly added ones, like NZ.
3. Visit the task list `http://tangaroa.test/wp-admin/admin.php?page=wc-admin&task=payments` and see WC Pay promoted.

<img width="696" alt="Screen Shot 2021-06-30 at 2 52 37 PM" src="https://user-images.githubusercontent.com/1922453/123894763-d839ba00-d9b2-11eb-8442-f8ed9a1bf3a6.png">
